extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.text-formatter.js

```diff
@@ -5,8 +5,12 @@ define('ext.wikia.design-system.on-site-notifications.text-formatter', [
 		'use strict';
 
 		function TextFormatter() {
-			function bold(text) {
-				return text ? '<b>' + text + '</b>' : text;
+			function escape(text) {
+				return text ? window.mw.html.escape(String(text)) : '';
+			}
+
+			function escapeAndBold(text) {
```

Comment:
I personally don't like too complex method. What do you think about using `bold(escape(text))`?